### DISCUSSION TITLE
Fill the helperText() with model Field Content

### DISCUSSION THREAD

**Discussion by SDJeff** [filamentphp/filament#12037] [Help] [answered]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

Can anybody please tell me how to get data of the the current record in a form and display it at the helperText() for example without creating any of the fields?

`Forms\Components\TextInput::make('catalog_number')
                    ->helperText($this->catalog->name)
                    ->required()
                    ->numeric(),`

Not sure perhaps it is to easy that i can find it in the documentation.

**Comment by leandrocfe** [comment, marked as answer]
```php
use Illuminate\Database\Eloquent\Model;

Forms\Components\TextInput::make('catalog_number')
    ->helperText(fn (?Model $record) => $record?->catalog_name)
    ->required()
    ->numeric()
```
this?
